src/main/java/rx/schedulers/Schedulers.java

```diff
@@ -137,4 +138,52 @@ public static TestScheduler test() {
     public static Scheduler from(Executor executor) {
         return new ExecutorScheduler(executor);
     }
+    
+    /**
+     * Starts those standard Schedulers which support the SchedulerLifecycle interface.
+     * <p>The operation is idempotent and threadsafe.
+     */
+    public static void start() {
```

Comment:
The javadoc should probably state that this is only needed if restarting after having called `shutdown`. 

Though I'm curious as to why calling `start` is needed. Why not make it so `shutdown` resets it back to nothing running, but if anything is invoked it starts up again just like on a fresh JVM start?